routing/routes.go

```diff
@@ -25,27 +25,85 @@ func NewRoutes() (r *Routes) {
 	r = new(Routes)
 
 	r.Routes = []Route{
+		// swagger:route GET / Root
+		// ---
+		// Handler to get the status of Kiali
+		//
+		//     Consumes:
+		//     - application/json
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http
+		// responses:
+		//      200: statusInfo
```

Comment:
I guess this "statusInfo" will be resolved by Swagger in the specific type. Isn't it ?

Reply:
Yes is like a tag this go to the swagger: response swagger:model called statusInfo